unstructured_inference/inference/layoutelement.py

```diff
@@ -8,6 +8,7 @@
 from pandas import DataFrame
 from PIL import Image
 
+from unstructured_inference.config import inference_config
 from unstructured_inference.constants import FULL_PAGE_REGION_THRESHOLD, SUBREGION_THRESHOLD_FOR_OCR
```

Comment:
Let me make sure I understand the convention... the constants in `unstructured_inference.constants` don't belong in `inference_config` because we don't want them to be overridden by environment variables. Is that correct?

Reply:
right; constants are variables that are NOT configurable. There is just one value we will use. While config contains values we want to tweak and may even need to change for different applications.
Let me add a comment to the module file to clarify